pyproject.toml

```diff
@@ -52,7 +52,7 @@ google = [
     "smart-open[gcs]>=5.2.1"
 ]
 snowflake = [
-    "apache-airflow-providers-snowflake",
+    "apache-airflow-providers-snowflake==3.2.0",
```

Comment:
I prefer to keep `>=` instead of `==`

Reply:
Fine for me.

Reply:
The question is do we want to still allow `>=3.1.0` or `>=3.2.0`.

Reply:
My bad, we don't really need this pin. Removing it.